ls1intum/Artemis
# `Communication`: Replace discussion labels with communication

### Checklist
#### General



#### Client

### Motivation and Context
The communication feature of Artemis was previously named as 'Discussion' within the platform. To respond to the updates of the feature and to make it easier to refer to, we change the label 'Discussion' to 'Communication'.

### Description
Please note that the router link is not updated in this PR and left as 'discussion' like before, since the accessibility of Posts from announcement emails depend on it. The router link is planned to be updated within the semester break.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course with postings enabled


1. Log in to Artemis
2. Navigate to a Course Page
3. Make sure that 'Discussion' label is replaced with 'Communication' and that the page is still accessible when clicked.
4. Navigate to an Exercise or Lecture
5. Make sure the 'Discussion' label is replaced with 'Communication' and the tooltips are updated accordinly.
6. Go to course management page.
7. Make sure that 'Discussion' label is replaced with 'Communication' and that the page is still accessible when clicked.
8. Click on a course.
9.  Make sure that 'Discussion' label is replaced with 'Communication' and that the page is still accessible when clicked.

### Documentation
https://artemis-platform--5289.org.readthedocs.build/en/5289/

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
Not changed.

### Screenshots
![image](https://user-images.githubusercontent.com/67344158/175789372-e8fe85fe-3543-4f38-b386-858d1fb4d8da.png)
![image](https://user-images.githubusercontent.com/67344158/175789376-bad39439-e632-4ade-9314-d0de00ac930b.png)
![image](https://user-images.githubusercontent.com/67344158/175789378-7d5516b2-f415-4e49-9711-f104059ffc63.png)
![image](https://user-images.githubusercontent.com/67344158/175789381-048d6a42-7689-47a4-8891-8b29912b6357.png)
![image](https://user-images.githubusercontent.com/67344158/175789383-ca195bd8-37b1-4f47-adac-5f0ba7eb00cf.png)
![image](https://user-images.githubusercontent.com/67344158/175789386-d1395a7e-ed21-4d59-ba9b-ec52d9daabc2.png)
![image](https://user-images.githubusercontent.com/67344158/175913748-e20d4839-0afc-44ef-953e-1d9c1efa0c6f.png)
![image](https://user-images.githubusercontent.com/67344158/175913978-ebcb1382-d59d-4107-8672-0679c797ef20.png)
![about us](https://user-images.githubusercontent.com/67344158/176118856-89cbe59f-cee7-4815-8ae9-aca2b3fe17a7.png)
